ext/opentelemetry-ext-aiohttp-client/tests/test_aiohttp_client_integration.py

```diff
@@ -0,0 +1,355 @@
+# Copyright 2020, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import contextlib
+import typing
+import unittest
+import unittest.mock
+import urllib.parse
+from http import HTTPStatus
+
+import aiohttp
+import aiohttp.test_utils
+import yarl
+
+import opentelemetry.ext.aiohttp_client
+from opentelemetry import trace
+from opentelemetry.sdk.trace import TracerProvider, export
+from opentelemetry.sdk.trace.export.in_memory_span_exporter import (
+    InMemorySpanExporter,
+)
+from opentelemetry.trace.status import StatusCanonicalCode
+
+
+class TestAioHttpIntegration(unittest.TestCase):
```

Comment:
This could be refactored to use `TestBase` https://github.com/open-telemetry/opentelemetry-python/blob/c074853a7cf9177847017c7656a2ccce461b191a/tests/util/src/opentelemetry/test/test_base.py#L26

Reply:
updated